cns/networkcontainers/networkcontainers_windows.go

```diff
@@ -136,7 +142,10 @@ func createOrUpdateWithOperation(createNetworkContainerRequest cns.CreateNetwork
 
 	log.Printf("[Azure CNS] Going to create/update network loopback adapter: %v", args)
 	c := exec.Command("cmd", args...)
```

Comment:
Shouldn't we lock before this line?

Reply:
nope, that just constructs the command, it's actually run in c.OutPut()